src/rollup/types.d.ts

```diff
@@ -38,6 +38,16 @@ export interface RollupLogProps {
 	url?: string;
 }
 
+export interface ExistingDecodedSourceMap {
+	file?: string;
+	mappings: number[][][];
```

Comment:
cf. my comment in collapseSourceMaps, if we introduce our own named types, this should use the mappings type as well.